tests/files/v1.0.2.config.json

```diff
@@ -0,0 +1,47 @@
+{
+    "nameserver": "",
+    "max retry": 3,
+    "log directory": "/root/.cvdupdate/logs",
```

Comment:
I get a test failure when i run this on my personal user account rather than in a docker container:
```
❯ python3 -m pytest .
======================================================= test session starts =======================================================
platform linux -- Python 3.8.10, pytest-4.6.9, py-1.8.1, pluggy-0.13.0
rootdir: /home/micasnyd/workspace/cvdupdate
collected 4 items

tests/test_cvdupdate.py ...F                                                                                                [100%]

============================================================ FAILURES =============================================================
____________________________________________ test_existing_state_migrates_successfully ____________________________________________

revert_homedir = None

    def test_existing_state_migrates_successfully(revert_homedir):
        ''' specifically test migrating an existing config.json to config + state.json'''
        default_cvdupdate_dir = Path.home() / '.cvdupdate/'
        default_cvdupdate_dir.mkdir(parents=True)

        # create a .cvdupdate/config.json which also contains dbs definitions
        old_config_file = 'tests/files/v1.0.2.config.json'
        old_config_json = json.loads(Path(old_config_file).read_text())

        shutil.copyfile(old_config_file, str(default_cvdupdate_dir / 'config.json'))

        # create cvdupdate object, which will read config.json and split state into state.json
>       a = CVDUpdate()

tests/test_cvdupdate.py:82:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cvdupdate/cvdupdate.py:126: in __init__
    self._init_logging()
cvdupdate/cvdupdate.py:147: in _init_logging
    if not self.log_dir.exists():
/usr/lib/python3.8/pathlib.py:1407: in exists
    self.stat()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = PosixPath('/root/.cvdupdate/logs')

    def stat(self):
        """
        Return the result of the stat() system call on this path, like
        os.stat() does.
        """
>       return self._accessor.stat(self)
E       PermissionError: [Errno 13] Permission denied: '/root/.cvdupdate/logs'

/usr/lib/python3.8/pathlib.py:1198: PermissionError
=============================================== 1 failed, 3 passed in 0.24 seconds ================================================
```

Interestingly, I can get past this error if I change:
```
❯ git diff
diff --git a/tests/files/v1.0.2.config.json b/tests/files/v1.0.2.config.json
index 9504cf7..3e59019 100644
--- a/tests/files/v1.0.2.config.json
+++ b/tests/files/v1.0.2.config.json
@@ -1,10 +1,10 @@
 {
     "nameserver": "",
     "max retry": 3,
-    "log directory": "/root/.cvdupdate/logs",
+    "log directory": "$HOME/.cvdupdate/logs",
     "rotate logs": true,
     "# logs to keep": 30,
-    "db directory": "/root/.cvdupdate/database",
+    "db directory": "$HOME/.cvdupdate/database",
     "rotate cdiffs": true,
     "# cdiffs to keep": 30,
     "dbs": {
@@ -40,8 +40,7 @@
             "CDIFFs": [
                 "bytecode-333.cdiff"
             ]
-       }
+        }
     },
     "uuid": "d93cd3bb-b2ee-4e6c-995b-aee50f7c3afb"
-}
-
+}
\ No newline at end of file
```
* note that my editor's format-on-save also tidied up a bit of the JSON.

Reply:
I went ahead and fixed this myself since I'm eager to see this feature merged.  
Thanks again for your work on this @bsanders